lib/FusionInventory/Agent/SNMP/Device.pm

```diff
@@ -53,15 +53,15 @@ my $inventory_only_base_variables = {
         type => 'count',
     },
     MEMORY       => {
-        oid  => '.1.3.6.1.2.1.25.2.3.1.5.1',
+        oid  => {'.1.3.6.1.2.1.25.2.3.1.5.1' => 'bytes'},
```

Comment:
I don't see why you're moving that oid in a hash a `bytes` should still be the default later in the updated algorithm.

Reply:
It was logic when I commited, but now, not really, I fix it...